MOBILE_API.xml

```diff
@@ -8236,7 +8236,7 @@
                 This parameter is filled for supporting OEM proprietary data exchanges.
             </description>
         </param>
-        <param name="url" type="String" maxlength="1000" mandatory="false">
+        <param name="url" type="String" mandatory="false">
```

Comment:
```suggestion
        <param name="url" type="String" mandatory="false" since="6.1">
```

Reply:
Also please add the history tag

```xml
<history>
  <param name="url" type="String" maxlength="1000" mandatory="false" since="3.0" until="6.1">
</history>
```

Reply:
@ShobhitAd 
Fixed in:https://github.com/smartdevicelink/rpc_spec/pull/225/commits/e7c8c5036d9d0eb43b83c8f8aa3f824716ca9588